src/classes/UTIL_ACCT_Naming.cls

```diff
@@ -35,296 +35,647 @@
 */
 public class UTIL_ACCT_Naming {
```

Comment:
If you haven't already done so, can you run the Apex PMD task on this class and see if it comes up with anything else to fix/address?

Reply:
Yep, I ran it again and fixed some of the variable capitalization issues. The other ones I'm getting is cyclomatic complexity, no viable alternative at input for the lines where it says while().